test/test_crossinheritance.py

```diff
@@ -818,6 +825,7 @@ def z(self):
         assert a.m_2 == 42
         assert a.m_3 == 67
 
+    @mark.xfail(run=not IS_CLANG_DEBUG, reason="Crashes with ClangRepl with 'toString not implemented'")
```

Comment:
I suspect the xfail tag here is wrong.
It should be
```suggestion
    @mark.xfail(run=False, condition=not IS_CLANG_DEBUG, reason="Crashes with ClangRepl with 'toString not implemented'")
```
Also, if this function passes in all other platforms, then the reason might not be related to `toString`.